manuscript/portal_iterative_forecasting_ms.Rmd

```diff
@@ -86,13 +86,13 @@ Once data are available, they must be processed into a form appropriate for mode
 
-Forecasts are currently evaluated using root mean square error (RMSE) to evaluate point forecasts and coverage to evaluate uncertainty. We plan to add additional metrics in the future. In addition to evaluating the actual forecasts, we also use hindcasting [forecasting on already collected data; @jolliffe2003] to gain additional insight into the methods that work best for forecasting this system. For example, a model is fit using rodent observations up to June 2005, then used to make a forecast 12 months out to May 2006. The observations of that 12-month period can immediately be used to evaluate the model. Since hindcasting is conducted using data that has already been collected, it allows model comparisons to be conducted on large numbers of hindcasts and provides insight into which models make the best forecasts without needing to wait for new data to be collected [@harris2018]. It can also be used to quickly evaluate new models instead of waiting for an adequate amount of data to accumulate. As the performance of different models is understood through evaluation of forecasts and hindcasts, models can be refined or removed from the system or ensemble to iteratively improve the resulting forecasts. 
+Forecasts are currently evaluated using root mean square error (RMSE) to evaluate point forecasts and coverage to evaluate uncertainty. We plan to add additional metrics, like deviance, that incorporate both accuracy and uncertainty [@hooten2015; @dietze2018]. In addition to evaluating the actual forecasts, we also use hindcasting [forecasting on already collected data; @jolliffe2003] to gain additional insight into the methods that work best for forecasting this system. For example, a model is fit using rodent observations up to June 2005, then used to make a forecast 12 months out to May 2006. The observations of that 12-month period can immediately be used to evaluate the model. Since hindcasting is conducted using data that has already been collected, it allows model comparisons to be conducted on large numbers of hindcasts and provides insight into which models make the best forecasts without needing to wait for new data to be collected [@harris2018]. It can also be used to quickly evaluate new models instead of waiting for an adequate amount of data to accumulate. As the performance of different models is understood through evaluation of forecasts and hindcasts, models can be refined or removed from the system or ensemble to iteratively improve the resulting forecasts. 
```

Comment:
this doesn't quit address Reviewer 3 comments about RMSE not being proper. Maybe something like:

We plan to add additional metrics, like deviance, that incorporate both accuracy and uncertainty and _better match the calibration method_ [@hooten2015; @dietze2018].